### ISSUE TITLE
[Bug]: Accessibility: all short name buttons read as "unlabeled" with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [meshtastic/Meshtastic-Android#2648]
### What happened?

All short name node chips read as unlabeled when using the TalkBack screen reader for Android.

<img width="234" height="2400" alt="Image" src="https://github.com/user-attachments/assets/b683f532-7c57-422e-9e89-82caea7aa6c7" />

### App Version

2.6.33

### Phone

Oneplus nord n200

### Device

Na

### Firmware

Na

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by DaneEvans** [comment]
Add it to #2259, and include which page (connections, nodes, etc. ) , unless it's all of them. 
That chip is common across the app. 

If you also include what you think it should be labeled, that will also see these issues get traction faster.

**Comment by shortwavesurfer2009** [comment]
It's all of them as far as I can tell. It's definitely in the nodes page and it's definitely in the chat screens when you are in a channel or private message. As far as what they label should be, it should just be what you see visually. So, whatever the short name says.
